mypy.ini

```diff
@@ -0,0 +1,8 @@
+# Global options:
+
+[mypy]
+disable_error_code = import
+show_error_codes = True
+exclude = (?x)(
+    build/lib/
+  )
```

Comment:
Some more options that can be helpful:
```
warn_redundant_casts = True
warn_unused_ignores = True
```
As mypy improved, this helped me to find which casts and ignores became unnecessary, and could be removed.
```
check_untyped_defs = True
```
To ensure that all functions are annotated.

Reply:
We could also name this `.mypy.ini` since most of the other config files also are hidden with a dot. mypy recognizes both.

Reply:
Thanks a lot for the suggestions, implemented them.